src/components/InfoPanel/InfoPanel.js

```diff
@@ -53,7 +54,13 @@ const InfoPanel = () => {
           <Topology width={300} height={300} modelData={modelStatusData} />
           <i
             className="p-icon--expand"
-            onClick={() => setShowExpandedTopology(!showExpandedTopology)}
+            onClick={() => {
+              setShowExpandedTopology(!showExpandedTopology);
+              ReactGA.event({
```

Comment:
This is triggering that it expanded even if it's closing. You'll want to add a conditional here.

Reply:
Negative - the close button is different.